fiorella/src/styles/main.scss

```diff
@@ -0,0 +1,29 @@
+@charset "UTF-8";
+/*============================================================================
+  [replace with theme name] | Built with Slate
+    - You cannot use native CSS/Sass @imports in this file without a build script
+==============================================================================*/
+
+/*================ ABSTRACTS ================*/
+@import url('abstracts/_variables.scss.liquid');
+@import url('abstracts/_mixins.scss');
+@import url('abstracts/_placeholders.scss');
+
+/*================ BASE ================*/
+@import url('base/_normalize.scss');
+
+/*================ LAYOUTS ================*/
+@import 'layout/_header.scss';
```

Comment:
Imported partial `layout/_header.scss` should be written as `layout/header`<br>Prefer double-quoted strings